src/controllerclientimpl.cpp

```diff
@@ -2070,4 +2070,62 @@ void ControllerClientImpl::ListFilesInController(std::vector<FileEntry>& fileent
     }
 }
 
+void ControllerClientImpl::CreateLogEntries(const std::vector<LogEntryPtr>& logEntries)
+{
+    if (logEntries.empty()) {
+        return;
+    } 
+
+    // uri to upload
+    _uri = _baseuri + std::string("api/v2/logEntry");
+
+    // prepare form
+    struct curl_httppost *formpost = NULL;
+    struct curl_httppost *lastptr = NULL;
+    CURL_FORM_RELEASER(formpost);
+
+    for (LogEntryPtr logEntry : logEntries) {
+        if (!logEntry) {
+            continue;
+        } 
+        // add log entry
+        std::string form_name = "logEntry/" + logEntry->logType;
+        curl_formadd(&formpost, &lastptr,
+                    CURLFORM_COPYNAME, form_name.c_str(),
+                    CURLFORM_COPYCONTENTS, DumpJson(logEntry->rEntry).c_str(),
```

Comment:
Seems pretty inefficient to allocate the serilized json string for each entry, Re-use a `rapidjson::StringBuffer`.